docs/cookbook/meetingPlanner.js

```diff
@@ -9,11 +9,11 @@ const timeZones = [
 ];
 
 // Start the table at midnight local time
-const calendarNow = now.inTimeZone(here);
+const calendarNow = now.toDateTime(here);
 const startTime = calendarNow
   .getDate()
-  .withTime(Temporal.Time.from('00:00')) // midnight
-  .inTimeZone(here);
+  .toDateTime(Temporal.Time.from('00:00')) // midnight
```

Comment:
After reading #700 I thought this might be clearer as `calendarNow.with(Temporal.Time.from('00:00'))` anyway?

Reply:
@ptomato didn't you update it? I'll take care of it while rebasing.

Reply:
Yes, it should be taken care of if you rebase.